src/providers/staticfile.rs

```diff
@@ -116,3 +124,21 @@ impl Provider for StaticfileProvider {
         ))))
     }
 }
+
+impl StaticfileProvider {
+    pub fn get_root(app: &App, env: &Environment, staticfile_root: String) -> String {
+        let mut root = "";
+        if env.get_variable("NIXPACKS_STATICFILE_ROOT").is_some() {
+            root = env.get_variable("NIXPACKS_STATICFILE_ROOT").unwrap();
+        } else if staticfile_root.is_empty() {
```

Comment:
Should this check if the root is not empty?

Reply:
Oops, My bad. Yeah, it should check if root is not empty.